backend/typescript/types.ts

```diff
@@ -1,12 +1,12 @@
-export type Role = number;
+export type Role = "User" | "Admin";
```

Comment:
This makes things easy with MongoDB since enums are stored as strings, but open to making this an actual Enum if that makes more sense for the Postgres implementation @zm2he